src/notation/inotationinteraction.h

```diff
@@ -82,12 +83,12 @@ class INotationInteraction
     virtual void clearSelection() = 0;
     virtual muse::async::Notification selectionChanged() const = 0;
     virtual void selectTopOrBottomOfChord(MoveDirection d) = 0;
+    virtual void notifyAboutSelectionChangedIfNeed() = 0;
```

Comment:
It should not be exposed in the interface. You can either pass NotationInteraction as an argument to NotationSelectionFilter(), or you can pass this function as a callback (which is probably better)